### ISSUE TITLE
Unable to connect to nextcloud after successful install

### ISSUE THREAD

**Issue by skyline6453** [danb35/freenas-iocage-nextcloud#214]
I'm very green at all of this.
I followed the instructions the best I knew how and bought my domain through cloudflare just before execution. I'm not sure if I have it misconfigured in cloudflare but after a successful install I am unable to connect to Nextcloud by either my domain address or by typing the IP of the jail locally
Jail_IP is the one my A cert on cloudflare dns is routing to, but thats private and I'm not sure if I should be pointing to a different one instead. 
Any help would be appreciated.

**Comment by David-Melik** [comment]
I confirm i have the same issue with duckdns

**Comment by skyline6453** [comment]
For a little more detail
I have tried prior to running the "iocage exec nextcloud /root/remove-staging.sh" and after. 
I have tried going direct to the jail IP on the local nextwork. 
I have tried SELFCERT and DNSCERT with an API from Cloudflare. 
Reinstalled multiple times. 
Re-imaged the Trunas install multiple times.
I have not been able to get to nextcloud

**Comment by tschettervictor** [comment]
You want to post your `nextcloud-config` file with sensitive content redacted?

**Comment by skyline6453** [comment]
I ended up reinstalling and trying to install via Plugin and was successful, but here is what my nextcloud-config file was that didn't work incase it helps anyone else.
JAIL_IP="192.168.0.XXX"
DEFAULT_GW_IP="192.168.0.1"
POOL_PATH="/mnt/stuff"
TIME_ZONE="America/Chicago"
HOST_NAME="mydomain.com"
STANDALONE_CERT=1
CERT_EMAIL="myemail@gmail.com"

I did find out that my domain was running through a proxy and it was braking, so that helps with external connections, but it doesn't really explain the failure to resolve to the jail IP in the network.

**Comment by tschettervictor** [comment]
> I ended up reinstalling and trying to install via Plugin and was successful, but here is what my nextcloud-config file was that didn't work incase it helps anyone else. JAIL_IP="192.168.0.XXX" DEFAULT_GW_IP="192.168.0.1" POOL_PATH="/mnt/stuff" TIME_ZONE="America/Chicago" HOST_NAME="mydomain.com" STANDALONE_CERT=1 CERT_EMAIL="[myemail@gmail.com](mailto:myemail@gmail.com)"
> 
> I did find out that my domain was running through a proxy and it was braking, so that helps with external connections, but it doesn't really explain the failure to resolve to the jail IP in the network.

Did you set your router DNS settings so that it resolves your domain to your local IP? If you ping your DNS locally, what does it resolve to?
You should not be using the plug-in.

**Comment by skyline6453** [comment]
Yes, and to me, the big area of concern was after multiple install attempts, it would say successful install, but even the Jail IP wouldn't resolve to anything on my local network. Even if the FQDN wasn't set up correctly, the local IP should still connect me to the Jail IP.

I do appreciate the concern and advice concerning the plug-in, but it is the only thing that has worked so far.

**Comment by tschettervictor** [comment]
> Yes, and to me, the big area of concern was after multiple install attempts, it would say successful install, but even the Jail IP wouldn't resolve to anything on my local network. Even if the FQDN wasn't set up correctly, the local IP should still connect me to the Jail IP.
> 
> I do appreciate the concern and advice concerning the plug-in, but it is the only thing that has worked so far.

This would probably be due to the script setting the domain variable in the config.php file under “trusted domains”

If you would set it to your IP instead of the domain, it should work.
